[2025-01-13T19:19:19.184+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-13T19:19:19.206+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_init.update_model_if_better manual__2025-01-13T18:26:44.261266+00:00 [queued]>
[2025-01-13T19:19:19.216+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_init.update_model_if_better manual__2025-01-13T18:26:44.261266+00:00 [queued]>
[2025-01-13T19:19:19.217+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-13T19:19:19.233+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): update_model_if_better> on 2025-01-13 18:26:44.261266+00:00
[2025-01-13T19:19:19.244+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=129) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-13T19:19:19.246+0000] {standard_task_runner.py:72} INFO - Started process 138 to run task
[2025-01-13T19:19:19.248+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'dag_init', 'update_model_if_better', 'manual__2025-01-13T18:26:44.261266+00:00', '--job-id', '25', '--raw', '--subdir', 'DAGS_FOLDER/craw_check_update.py', '--cfg-path', '/tmp/tmptg0w5ete']
[2025-01-13T19:19:19.249+0000] {standard_task_runner.py:105} INFO - Job 25: Subtask update_model_if_better
[2025-01-13T19:19:19.307+0000] {task_command.py:467} INFO - Running <TaskInstance: dag_init.update_model_if_better manual__2025-01-13T18:26:44.261266+00:00 [running]> on host 0d9f7ac256fb
[2025-01-13T19:19:19.405+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='dag_init' AIRFLOW_CTX_TASK_ID='update_model_if_better' AIRFLOW_CTX_EXECUTION_DATE='2025-01-13T18:26:44.261266+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-13T18:26:44.261266+00:00'
[2025-01-13T19:19:19.407+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-13T19:19:19.408+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-13T19:19:19.409+0000] {logging_mixin.py:190} INFO - Current task name:update_model_if_better state:running start_date:2025-01-13 19:19:19.207698+00:00
[2025-01-13T19:19:19.410+0000] {logging_mixin.py:190} INFO - Dag name:dag_init and current dag run status:running
[2025-01-13T19:19:19.410+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-13T19:19:19.412+0000] {logging_mixin.py:190} INFO - Not enough labeled data to train a new model.
[2025-01-13T19:19:19.413+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-13T19:19:19.424+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-13T19:19:19.425+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=dag_init, task_id=update_model_if_better, run_id=manual__2025-01-13T18:26:44.261266+00:00, execution_date=20250113T182644, start_date=20250113T191919, end_date=20250113T191919
[2025-01-13T19:19:19.436+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-13T19:19:19.437+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-13T19:19:19.438+0000] {logging_mixin.py:190} INFO - Dag name:dag_init queued_at:2025-01-13 19:17:06.742287+00:00
[2025-01-13T19:19:19.439+0000] {logging_mixin.py:190} INFO - Task hostname:0d9f7ac256fb operator:PythonOperator
[2025-01-13T19:19:19.468+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-13T19:19:19.488+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-13T19:19:19.491+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
